MovieDB/MovieDB/Scenes/Dashboard/DashboardViewController.swift

```diff
@@ -0,0 +1,123 @@
+import UIKit
+import RxSwift
+import RxCocoa
+import Reusable
+import NSObject_Rx
+import Then
+
+fileprivate enum ConstantDashboard {
+    static let movieListTableViewRowHeight = CGFloat(138)
+    static let fontSizeCategoryTitle = CGFloat(12)
+}
+
+final class DashboardViewController: UIViewController, StoryboardBased, Bindable {
+    @IBOutlet private weak var categoryListCollectionView: UICollectionView!
+    @IBOutlet private weak var movieListTableView: UITableView!
+    @IBOutlet private weak var searchTextField: UITextField!
+    @IBOutlet private weak var searchButton: UIButton!
+    @IBOutlet private weak var watchListButton: UIButton!
+    @IBOutlet private weak var loadMoreButton: UIButton!
+    
+    var viewModel: DashboardViewModel!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configView()
+    }
+    
+    func bindViewModel() {
+        let input = DashboardViewModel
+            .Input(loadTrigger: Driver.just(()),
+                   searchButtonTrigger: searchButton.rx.tap.asDriver(),
+                   watchListButtonTrigger: watchListButton.rx.tap.asDriver(),
+                   loadMoreButtonTrigger: loadMoreButton.rx.tap.asDriver(),
+                   searchText: searchTextField.rx.text.orEmpty.asDriver(),
+                   categorySelectedTrigger: categoryListCollectionView.rx.modelSelected(CategoryType.self).asDriver(),
+                   movieSelectedTrigger: movieListTableView.rx.modelSelected(Movie.self).asDriver())
+        
+        let output = viewModel.transform(input, disposeBag: rx.disposeBag)
+        
+        output.categoryItems
+            .asDriver()
+            .drive(categoryListCollectionView.rx.items) { collectionView, index, item in
+                let indexPath = IndexPath(item: index, section: 0)
+                let cell: CategoryItemCollectionViewCell = collectionView.dequeueReusableCell(for: indexPath)
+                cell.viewModel = CategoryItemCollectionViewCellViewModel(category: item)
+                return cell
+            }
+            .disposed(by: rx.disposeBag)
+        
+        output.movieItems
+            .asDriver()
+            .drive(movieListTableView.rx.items) { tableView, index, item in
+                let indexPath = IndexPath(item: index, section: 0)
+                let cell: MovieItemTableViewCell = tableView.dequeueReusableCell(for: indexPath)
+                cell.viewModel = MovieItemTableViewCellViewModel(movie: item)
+
+                //MARK: - Bookmark Button Action
+                cell.bookmarkButton.rx.tap.asDriver()
```

Comment:
Same as above comment. Don't access to UI of the cell at outside.
Logic of action can be handled inside of cell, it should be fine